tests/test_batches/test_chunked/test_parquet.py

```diff
@@ -9,6 +13,27 @@
 )
 
 
+@pytest.fixture
+def local_writable_url(tmpdir):
+    yield f"file://{tmpdir.mkdir('data')}"
+
+
+@pytest.fixture
+def s3_writable_url():
+    test_bucket_name = "test_bucket"
+    with moto.mock_s3():
+        client = boto3.client("s3", "us-east-1")
+        client.create_bucket(Bucket=test_bucket_name)
+        url = f"s3://{test_bucket_name}"
+        yield url
+
+
+def _ddf_to_parquet(ddf, path):
+    for i, df in enumerate(ddf.partitions):
+        with tio.open(f"{path}/chunk_{i}.parquet", "wb") as fh:
+            df.compute().to_parquet(fh)
```

Comment:
i noticed that timeserio was still depending on s3fs via the parquet batch generator that wasn't caught by tests, hence the additions here: adding to the tests to check writing to s3, and moving the mock data creation from s3fs to pure tentaclio